### PR TITLE
fix: mobile layout for Connect Modal

### PR DESCRIPTION
**Pull request by totraev**
### Major changes

- was fixed header and footer sections to the top and bottom of the modal correspondingly
- added `tailwind-merge` lib
- added `postcss-import` plugin for postcss and fixed issue with global styles

### Before
Screenshot 2024-09-10 at 17 56 23

Screenshot 2024-09-10 at 17 56 41

### After
Screenshot 2024-09-10 at 17 55 15

Screenshot 2024-09-10 at 17 54 49

### CODE REVIEW

File: src/app/components/Modals/GeneralModal.tsx [hunk 1]
```diff
@@ -43,7 +45,7 @@ export const GeneralModal: React.FC<GeneralModalProps> = ({
       onClose={() => onClose(false)}
       classNames={{
         modalContainer: "flex items-end justify-center md:items-center",
-        modal: `m-0 w-full max-w-none rounded-t-2xl bg-base-300 shadow-lg md:w-auto md:rounded-b-2xl max-h-[85svh] min-w-[20rem] md:min-w-[30rem] ${getSize()}`,
+        modal: `m-0 w-full max-w-none rounded-t-2xl bg-base-300 shadow-lg md:w-auto md:rounded-b-2xl max-h-[100svh] min-w-[20rem] md:max-h-[85svh] md:min-w-[30rem] ${getSize()} ${className}`,
```
Comment (totraev) [inline src/app/components/Modals/GeneralModal.tsx:48]: @gbarkhatov I'd recommend to consider this lib for tailwind's classNames

**Comment by gbarkhatov** [comment]
@totraev please add
- description
- screenshots (before an after) for such PRs so it's easier to navigate later on

**Review by gbarkhatov — APPROVED** [review]

**Comment by gbarkhatov** [comment]
@totraev if you plan to use `tailwind-merge` @ ping me to re-review

File: src/app/components/Modals/ConnectModal.tsx [hunk 2]
```diff
@@ -110,7 +111,10 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
     return (
       <button
         key={name}
-        className={`flex cursor-pointer items-center gap-2 rounded-xl border-2 bg-base-100 p-2 transition-all hover:text-primary ${selectedWallet === BROWSER ? "border-primary" : "border-base-100"}`}
+        className={twJoin(
```
Comment (totraev) [inline src/app/components/Modals/ConnectModal.tsx:114]: @gbarkhatov here

**Review by jeremy-babylonlabs — APPROVED** [review]

**Pull request merged**